src/rgbToHsv.jsx

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
It should has .js extension

Reply:
with the extension .js webpack did not compile the project (did not find this file)

Reply:
Actually, it sounds strange because hsvToRGB has the .js extension

Reply:
I know that it's strange, maybe a Windows?

Reply:
client?8505:167 ./src/rgbToHsv.jsx
Module build failed: Error: ENOENT: no such file or directory, open 'D:\KODING\issue\react-circular-color\src\rgbToHsv.jsx'
 @ ./src/index.jsx 25:17-38

Reply:
hmm, now all ok